src/lib/server.ts

```diff
@@ -72,7 +72,7 @@ export class Client {
         this
       ),
     [Instance.node]: (message) =>
-      NodeMessageHandler.handle(
+      this.clientsController.nodeMessageHandler.handle(
```

Comment:
Why does `nodeMessageHandler` live on `clientsController` while all other handlers are defined on the server?